client/src/types.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+export type PlaceProps = {|
+  name: string,
```

Comment:
are all the props mandatory?
what if I'll want to add/update place without filling all the props (ex: no lat/lon)?
does the server validates all these fields?

Reply:
To make things simple, I'm planning to use a single form for both `create` and `update`, so all fields are sent to the server even upon update (which helps in case someone wants to delete an attribute).
Server does validation.

Reply:
Minor: maybe you can utilize it more, what about the rendered component that’ll use this?. Flow will shout on missing props/why pass empty/undefined ones...
If we’re going for all props then there should also be a score/rating function evaluator or even an array of ReviewProps